test/src/mistralai_client_dart_base_chat_test.dart

```diff
@@ -247,9 +246,7 @@ const chatCompletionParamsWithAllFieldsBody = '''
   "messages": [
     {
       "role": "user",
-      "content": "What is the best French cheese?",
-      "name": null,
-      "tool_calls": null
+      "content": "What is the best French cheese?"
```

Comment:
Ok so the name of the const is now not reflecting the actual body.

Reply:
Resolved in [e824818](https://github.com/nomtek/mistralai_client_dart/pull/45/commits/e824818379c0768111aefed6a10c237da468dd83)